pack/apk.mk

```diff
@@ -39,11 +44,15 @@ package: $(BUILDDIR)/$(TARBALL) \
 	@echo "Building APK packages"
 	@echo "-------------------------------------------------------------------"
 	sudo apk update
+	mkdir ~/.abuild/
+	echo "PACKAGER_PRIVKEY=/home/${USER}/.abuild/private.rsa" > ~/.abuild/abuild.conf
+	echo "$(ABUILD_KEY)" | base64 -d > ~/.abuild/private.rsa
 	rm -f $(BUILDDIR)/*/APKINDEX.tar.gz
 	rm -rf $(BUILDDIR)/src
 	cd $(BUILDDIR); abuild checksum
 	cd $(BUILDDIR); abuild -r -P $(BUILDDIR) -s $(BUILDDIR)
 	mv -f $(BUILDDIR)/*/*.apk $(BUILDDIR)
+	tar -czf $(BUILDDIR)/keys.tgz -C /home/${USER}/.abuild/ .
```

Comment:
Is it useful, when a caller is already obligated to have the key? Or it was made for a previous version of the patch, when the keys were autogenerated and useless now? If the latter is true, I would remove it. Less possibilities to make a mistake and reveal the private key.